### ISSUE TITLE
Crash with last versions Forestry unstable

### ISSUE THREAD

**Issue by Neuromantix** [ExtrabiomesXL/ExtrabiomesXL#436]
http://pastebin.com/gYwhBgkG

**Comment by allaryin** [comment]
```
Caused by: java.lang.NoSuchFieldError: plainFlowers
        at extrabiomes.plugins.ForestryPlugin.addFlowers(ForestryPlugin.java:151)
        at extrabiomes.plugins.ForestryPlugin.postInit(ForestryPlugin.java:95)
```

Catch NoSuchFieldError's when loading plugins to disable integration when people break their own API's with future updates.

**Comment by Neuromantix** [comment]
Forestry team answer:
https://github.com/ForestryMC/ForestryMC/issues/783

**Comment by ScottKillen** [comment]
As reported at https://github.com/ForestryMC/ForestryMC/issues/783, Forestry chose to break its own API along with compatibility with every mod using it, they could have chosen to be backwards compatible, but they have not. This is a huge change in position since @SirSengir first wrote the Forestry compatibility code for Extrabiomes. Basically, the Forestry devs are saying: "We messed everything up. Too bad. Go talk to the other guy. It is their fault that we are not going to fix it."

We sympathize. It is unfortunate that the Forestry devs have put you in this position. Relative to the Forestry team, we have very limited development resources...and the truth is that the Forestry devs, if they cared about this issue, could easily submit a patch for Extrabiomes as a [pull request](https://github.com/ExtrabiomesXL/ExtrabiomesXL/pulls). They are choosing to redirect your frustrations to us rather than own the design choice they made that put you in this situation.

**Comment by mezz** [comment]
Relax, before going hostile just contact me. I'm happy to make a PR if you need me to.

**Comment by ScottKillen** [comment]
Friend, there is a vast distance between hostility and frustrated disappointment.

**Comment by allaryin** [comment]
A build containing this fix is now uploaded to Curse as 3.16.3 (http://minecraft.curseforge.com/mc-mods/60041-extrabiomesxl/files/2252754).
